clients/ui/bff/internal/integrations/kubernetes/client.go

```diff
@@ -2,13 +2,13 @@ package kubernetes
 
 import (
 	"context"
+
 	corev1 "k8s.io/api/core/v1"
 )
 
 const ComponentLabelValue = "model-registry"
 const ComponentLabelValueCatalog = "model-catalog"
 
-// TODO ppadti double check if the config map key is indeed sources.yaml
 const CatalogSourceKey = "sources.yaml"
 const CatalogSourceDefaultConfigMapName = "model-catalog-source-config"
```

Comment:
@ppadti the default configmap is called `model-catalog-default-sources`. this is causing a 500 error after the sync to ODH.